src/Raven.Client/Documents/Session/Tokens/OrderByToken.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text;
+using Raven.Client.Documents.Queries;
+
+namespace Raven.Client.Documents.Session.Tokens
+{
+    public class OrderByToken : QueryToken
```

Comment:
90% of the tokens is singletons, and IIRC rest are immutable (need to remove Clone method, not needed I think)